posts/zzz_DO_NOT_EDIT_introducing.../introducing_docorator.qmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "Introducing {docorator} to the Pharmaverse"
+author:
+  - name: Becca Krouse
+description: "A framework for production-ready outputs"
+# Note that the date below will be auto-updated when the post is merged.
+date: "2025-12-15"
+# Please do not use any non-default categories.
+# You can find the default categories in the repository README.md
+categories: [TLG]
+# Feel free to change the image
+image: "docorator_hex.png"
+
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "zzz_DO_NOT_EDIT_introducing..."
+# library(link)
+# link::auto()
```

Comment:
@manciniedoardo these are commented out so the link package won't make the auto-links for packages and functions anymore.